DB/src/main/java/io/deephaven/db/v2/by/ComboAggregateFactory.java

```diff
@@ -1352,4 +1378,97 @@ private static void setLeafRollupAttributes(QueryTable table) {
         table.setAttribute(Table.HIERARCHICAL_CHILDREN_TABLE_MAP_ATTRIBUTE, TableMap.emptyMap());
         table.setAttribute(Table.REVERSE_LOOKUP_ATTRIBUTE, ReverseLookup.NULL);
     }
+
+    private static class ComboByAggregationAdapter implements Aggregation.Visitor {
+
+        private ComboBy out;
+
+        public ComboBy getOut() {
+            return Objects.requireNonNull(out);
+        }
+
+        static String asMatchPair(Aggregation match) {
+            return MatchPair.of(match.match()).toString();
```

Comment:
We turn it into a string so something else can re-parse it?

Reply:
Had to add some new constructors / methods in DB, but now we can translate straight-up.